### PR TITLE
PAN-OS running ***panorama-get-url-category*** multiple times present…

### PR DESCRIPTION
**Pull request by bakatzir**
…ed in the war room data from the context.

## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/24386

## Screenshots
in the issue

## Minimum version of Demisto
- [x] 3.5.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests - no, as we do not have the urlfiltering license in all of our instances.
- [x] Documentation

### CODE REVIEW

File: Packs/Legacy/Integrations/Panorama/Panorama.py [hunk 1]
```diff
@@ -2228,27 +2228,38 @@ def panorama_get_url_category_command(url_cmd: str):
     urls = argToList(demisto.args()['url'])
 
     categories_dict: Dict[str, list] = {}
+    categories_dict_hr: Dict[str, list] = {}
```
Comment (ronykoz) [inline Packs/Legacy/Integrations/Panorama/Panorama.py:2231]: not sure why do we need it if we have categories_dict

Comment (bakatzir) [reply Packs/Legacy/Integrations/Panorama/Panorama.py:2231]: I want to have a separate one for the HR, as `categories_dict` is used to populate the data from the context, so we can add new urls to already pre defined categories in the context.

**Comment by bakatzir** [comment]
Last build passed, files changed location as part of marketplace.

**Comment by bakatzir** [comment]
conflicts with changelog due to release, merging.

**Pull request merged**